report_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	"os"
 	"testing"
 
-	"github.com/izakmarais/reporter/grafana"
+	"github.com/JCN-9000/reporter/grafana"
```

Comment:
You need to revert these package path changes. Use the original path but create a remote entry in git to your fork.
cc: @IzakMarais

Reply:
I was afraid of this ... no idea how to do it though

Reply:
Tried and failed : I suppose I can't add ME as remote to this project and can't use this project as remote for mine, since I ca't push/write to it. Or am I understanding you wrong ? Local tests pass but travis fails.

Reply:
I guess Travis was overwriting some files because "izakmarais" != "IzakMarais" and did a go get over the already fetcher files of the PR.

Reply:
Hmm ok, can you elaborate on this? How is this case mismatch between the code and my git repo affecting the builds?

Reply:
When checking the logs from Travis this is what it does:
- git pull izakmarais:master into  IzakMarais/reporter  folder
- git pull patches merged into the same folder
then it starts the go part doing
- go get -t -v ./...    but this triggers the download of izakmarais/reporter in a different new folder which overrides the code in the patched folder, since the code references the packages like izakmarais/reporter

If the code references the packages as IzakMarais/reporter there is no download and the test goes on correctly.

I suppose GIT is managing repositories in a case-insensitive way, but GO is not.

Reply:
Thanks for the explanation. This seems like something I need to fix first, in since it is affecting all three open PRs.